### ISSUE TITLE
Refactorization recommended in apptest.py, status_file.py, and the harness utilities

### ISSUE THREAD

**Issue by hagertnl** [olcf/olcf-test-harness#84]
In the places using InfluxDB, there is a lot of common code that overlaps. I'd suggest creating a class that handles the InfluxDB transactions and is imported by apptest, status_file, and all harness utilities codes. A very simple API like:
```
influx_interface
    |- post_to_influx(post_body)
    |- run_flux_query(query)
    |- run_influxql_query(query)
```
would be all we need for a simple start.

**Comment by hagertnl** [comment]
The idea is that the `run_*_query` methods would return a similar data structure every time -- a list of dict objects. This would save ~100 lines of code in each of the relevant utilities.

**Comment by hagertnl** [comment]
Part of this is implemented in #87 , but the updates to the harness utility scripts have not been updated in that PR.

**Comment by hagertnl** [comment]
Closed by #177 .
